ale/drivers/base.py

```diff
@@ -339,50 +420,39 @@ def center_ephemeris_time(self):
         """
         if not hasattr(self, '_center_ephemeris_time'):
             halflines = self.image_lines / 2
-            center_sclock = self.starting_ephemeris_time + halflines * self._exposure_duration
+            center_sclock = self.starting_ephemeris_time + halflines * self.line_exposure_duration
             self._center_ephemeris_time = center_sclock
         return self._center_ephemeris_time
 
-    @property
-    def t0_ephemeris(self):
-        return self.starting_ephemeris_time - self.center_ephemeris_time
-
-    @property
-    def t0_quaternion(self):
-        return self.starting_ephemeris_time - self.center_ephemeris_time
 
+class Isis3():
     @property
-    def dt_ephemeris(self):
-        return (self.ending_ephemeris_time - self.starting_ephemeris_time) / self.number_of_ephemerides
+    def start_time(self):
+        return self._label['IsisCube']['Instrument']['StartTime']
 
     @property
-    def dt_quaternion(self):
-        return (self.ending_ephemeris_time - self.starting_ephemeris_time) / self.number_of_ephemerides
+    def spacecraft_name(self):
+        return self._label['IsisCube']['Instrument']['SpacecraftName']
 
     @property
-    def number_of_ephemerides(self):
-        return 909
+    def image_lines(self):
+        return self._label['IsisCube']['Core']['Dimensions']['Lines']
 
     @property
-    def number_of_quaternions(self):
-        return 909
-
-
+    def image_samples(self):
+        return self._label['IsisCube']['Core']['Dimensions']['Samples']
 
     @property
     def _exposure_duration(self):
-        return self._label['LINE_EXPOSURE_DURATION'].value * 0.001  # Scale to seconds
-
-class Framer(Base):
-
-    @property
-    def name_model(self):
-        return "USGS_ASTRO_FRAME_SENSOR_MODEL"
+        return self._label['IsisCube']['Instrument']['ExposureDuration'].value * 0.001 # Scale to seconds
```

Comment:
I wonder if we should have a `scale` attribute to pull the magic number from here? We would need a scale for each attribute that we cared about.

Reply:
I think this is something that we can work on as we add more supported instruments. Right now, I don't see enough need for this.